hooks/commands/run.sh

```diff
@@ -8,6 +8,7 @@ run_service="$(plugin_read_config RUN)"
 container_name="$(docker_compose_project_name)_${run_service}_build_${BUILDKITE_BUILD_NUMBER}"
 override_file="docker-compose.buildkite-${BUILDKITE_BUILD_NUMBER}-override.yml"
 pull_retries="$(plugin_read_config PULL_RETRIES "0")"
+shell="$(plugin_read_config SHELL "/bin/bash -c")"
```

Comment:
I think we have `-e` in the vanilla bootstrap command hook, to ensure that commands don't keep executing if one fails

Reply:
Could we just default to `/bin/sh`, which on most systems will just be `bash` anyway?

Reply:
Yup, I reckon your are totes correct on both calls. As always you are a fine human to collaborate with.